wp-content/themes/haruoshi_origin/archive-works.php

```diff
@@ -26,7 +26,12 @@
             </li>
             <?php endwhile; ?>
         </ul>
-        <?php endif;?>
+        <?php
+        if (function_exists("pagination")) {
+            pagination($additional_loop->max_num_pages);
```

Comment:
```suggestion
            pagination($wp_query->max_num_pages);
```
通常はこっちだと思う。
`$additional_loop` 初めて見た。

Reply:
done https://github.com/oooshima/origin_wordpress/pull/35/commits/9822e53daf153d4128192478ee7d39619fa0dde7